convert2rhel/actions/__init__.py

```diff
@@ -545,13 +555,28 @@ def find_actions_of_severity(results, severity):
 
     Example::
 
-        failed_actions = find_actions_of_severity(results, "SKIP")
-        # failed_actions will contain all actions which were skipped
+        matched_actions = find_actions_of_severity(results, "SKIP")
+        # matched_actions will contain all actions which were skipped
         # or failed while running.
     """
-    failed_actions = [a for a in results.items() if a[1]["status"] >= STATUS_CODE[severity]]
+    matched_actions = [a for a in results.items() if a[1]["status"] >= STATUS_CODE[severity]]
```

Comment:
Not to be too picky here, but could we use the full word instead of a single letter? What does `a` means here? Could be anything depending on the viewer

```suggestion
    matched_actions = [action for action in results.items() if action[1]["status"] >= STATUS_CODE[severity]]
```

I feel that using `action` would be way more readable rather than a single missing `a` in there